library/std/src/backtrace.rs

```diff
@@ -110,13 +104,15 @@ use crate::vec::Vec;
 /// previous point in time. In some instances the `Backtrace` type may
 /// internally be empty due to configuration. For more information see
 /// `Backtrace::capture`.
+#[stable(feature = "backtrace", since = "1.65.0")]
 #[must_use]
 pub struct Backtrace {
     inner: Inner,
 }
 
 /// The current status of a backtrace, indicating whether it was captured or
 /// whether it is empty for some other reason.
+#[stable(feature = "backtrace", since = "1.65.0")]
```

Comment:
I think these should be 1.64.0: https://github.com/rust-lang/rust/blob/master/src/version

Reply:
Fixed